### PR TITLE
Add a generic pre-setup checklist

### PR DESCRIPTION
**Pull request by tobru**

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+* In which account will the cluster be installed? Gather the needed access credentials with sufficient rights.
+
+Depending on what the infrastructure supports:
+
+* Which region and zone will be used for the cluster?
+* Do the worker nodes have to be spread over multiple availability zones (AZ)?
+
+== Network requirements
+
+Depending on the requirements to the network, the following information need to be prepared:
```
Comment (srueg) [inline docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc:48]:
```suggestion
Depending on the requirements to the network, the following information needs to be prepared:
```

File: docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+* What's the IP subnet for the nodes?
+* Which IP network can be used for `clusterNetwork` (SDN)? Should be a `/14`.
```
Comment (srueg) [inline docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc:51]: I'd suggest to turn these questions: "Is a change to the default network necessary?"

**Review by srueg — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc (hunk 2 above)
Comment (tobru) [reply docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc:51]: Rephrased, please check again.

**Review by srueg — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+The main purpose of the cluster will be to run the users applications and these applications have specific resource requirements.
+
+This needs to be taken into consideration when answering the following questions:
+
+* What kind of virtual machines will be needed to satisfy the resource requirements (vCPU / Memory) for running the user's workload?
+* How many user application Pods will be scheduled (just an estimate)?
+* How many worker nodes will be needed?
+* How much room for failure and maintenance (`n+1` redundancy) is needed?
+
+Don't go below the defined minimum requirements.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc:32]: Can we link here to the _defined minimum_?

**Review by corvus-ch — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc (hunk 3 above)
Comment (tobru) [reply docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc:32]: The issue here is that I don't know where to link to, as we define the minimum requirements individually per infrastructure under https://kb.vshn.ch/products/vss_ocp4.html and I didn't find a "generic" page on docs.openshift.com.

**Review by corvus-ch — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc (hunk 3 above)
Comment (corvus-ch) [reply docs/modules/ROOT/pages/how-tos/generic-pre-install-checklist.adoc:32]: I guess we have to figure that one out. But I am fine leave that for later.

**Pull request merged**